backend/src/v5/routes/users.js

```diff
@@ -280,7 +286,7 @@ const establishRoutes = () => {
 	* @openapi
 	* /user:
 	*   put:
-	*     description: Updates the profile of the logged in user
+	*     description: Updates the profile of the logged in user. SSO users cannot update their email, password, firstname and lastname
```

Comment:
i think that should be specified in the description of those individual properties (~L300+)